modules/apps/object/object-service/src/main/java/com/liferay/object/internal/notification/term/contributor/ObjectDefinitionNotificationTermEvaluator.java

```diff
@@ -116,6 +172,44 @@ private Map<String, Long> _getObjectFieldIds() {
 		return objectFieldIds;
 	}
 
+	private Map<String, String> _getUserTermValuesMap(
+		Context context, User user) {
+
+		Contact contact = user.fetchContact();
+
+		return HashMapBuilder.put(
+			"CREATOR",
+			() -> context.equals(Context.RECIPIENT) ?
+				String.valueOf(user.getUserId()) : user.getFullName(true, true)
+		).put(
+			"EMAIL", user.getEmailAddress()
+		).put(
+			"FIRSTNAME", user.getFirstName()
+		).put(
+			"ID", String.valueOf(user.getUserId())
+		).put(
+			"LASTNAME", user.getLastName()
+		).put(
+			"MIDDLENAME", user.getMiddleName()
+		).put(
+			"PREFIX",
+			() -> (contact == null) ? StringPool.BLANK :
+				_getListTypeName(contact.getPrefixListTypeId())
+		).put(
+			"SUFFIX",
+			() -> (contact == null) ? StringPool.BLANK :
+				_getListTypeName(contact.getSuffixListTypeId())
+		).build();
+	}
+
+	private static final Set<String> _currentUserTermName =
+		Collections.unmodifiableSet(
+			SetUtil.fromArray(
+				"[%CURRENT_USER_EMAIL%]", "[%CURRENT_USER_FIRSTNAME%]",
+				"[%CURRENT_USER_ID%]", "[%CURRENT_USER_LASTNAME%]",
+				"[%CURRENT_USER_MIDDLENAME%]", "[%CURRENT_USER_PREFIX%]",
+				"[%CURRENT_USER_SUFFIX%]"));
```

Comment:
In the enum of the previously task, we have two informations **"key, termName".**
Here I need only the **termName**. Make sense use enum here too?

Reply:
To avoid contaminate this PR with changes that belongs to the first task PR(not merged yet), I was gather both contents in another branch and sent a private PR with this requested change: https://github.com/danvb112/liferay-portal/pull/41